### ISSUE TITLE
[Custom] Double XP weekend buff 

### ISSUE THREAD

**Issue by Echinaceax** [WoWManiaUK/Blackwing-Lair#4854]
**Suggestion approved** 

**What should happen:**
Set a buff when weekend xp is active like in mania server!!!

**Comment by mfranca0009** [comment]
We need a client patch for WoW-Reign to bring the updated spell for spell id 62121 to be renamed correctly, then I should be able to figure out how to add this buff when the event is active.

**Comment by Echinaceax** [comment]
@mfranca0009 
Yes i asked Jani about it and he said its something he has to do 
forgot to update here
